### DISCUSSION TITLE
Calling server endpoints locally instead of via HTTP Requests

### DISCUSSION THREAD

**Discussion by DoisKoh** [sveltejs/kit#9456] [General]
Is there already a pattern that exists for this? This would bring some performance improvements by localizing web server to web server calls, but it also allows me to greatly simplify/skip handling extra authentication (I could just read from locals). 

Essentially, this allows me to reuse the endpoint function... instead of having to abstract them into some async utility function elsewhere, and then call the endpoint only when it's from the browser, and call the function from my library only when it's on the server.

**Comment by geoffrich** [comment]
If you're calling the server endpoint in a `load` function, using the provided `fetch` will call the handler directly instead of making an HTTP request.

> internal requests (e.g. for +server.js routes) go direct to the handler function when running on the server, without the overhead of an HTTP call

https://kit.svelte.dev/docs/load#making-fetch-requests
